crates/neon/build.rs

```diff
@@ -0,0 +1,13 @@
+use std::{env, fs, path::PathBuf};
+
+fn main() {
+    let out_path = PathBuf::from(env::var("OUT_DIR").unwrap());
+    let out_file = out_path.join("type_tag.rs");
+    let (lower, upper) = rand::random::<(u64, u64)>();
```

Comment:
We should only generate 64-bits of randomness and use `0` for `upper`. This gives us some reserved space to change the scheme at a later date.